Backend/app/rider-platform/rider-app/Main/src/Domain/Types/Merchant/MerchantOperatingCity.hs

```diff
@@ -0,0 +1,29 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module Domain.Types.Merchant.MerchantOperatingCity where
+
+import qualified Domain.Types.Merchant as DM
+import Kernel.Prelude
+import Kernel.Types.Id
+
+data MerchantOperatingCity = MerchantOperatingCity
+  { id :: Id MerchantOperatingCity,
+    merchantId :: Id DM.Merchant,
+    city :: City
+  }
+  deriving (Generic, FromJSON, ToJSON, Show, Eq, ToSchema)
+
+data City = BANGALORE | KOLKATA | KOCHI | CHENNAI | DELHI | MUMBAI | DEFAULT
```

Comment:
Instead of this can you use the City type defined in Shared Kernel for Beckn Context as we can keep the City common in the code and we are using that City only to pass in Search Request from BAP to BPP. It will be better to reuse the same types.